engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -186,4 +209,30 @@
 	 * @since 5.2
 	 */
 	HibernateValidatorConfiguration timeProvider(TimeProvider timeProvider);
+
+	/**
+	 * Allow to set a {@code MethodValidationConfiguration} to modify the behavior of the Validator with respect to
+	 * Specification section 4.5.5. In particular:
+	 * <pre>
+	 * "Out of the box, a conforming Bean Validation provider must throw a
+	 * ConstraintDeclarationException when discovering that any of these rules are violated.
+	 * In addition providers may implement alternative, potentially more liberal, approaches
+	 * for handling constrained methods in inheritance hierarchies. Possible means for activating
+	 * such alternative behavior include provider-specific configuration properties or annotations.
+	 * Note that client code relying on such alternative behavior is not portable between Bean
+	 * Validation providers."
+	 * </pre>
+	 *
+	 * @param mvc {@code MethodValidationConfiguration} instance with the method validation behaviour configuration
+	 * settings.
+	 *
+	 * @return {@code this} following the chaining method pattern
+	 */
+	HibernateValidatorConfiguration setMethodValidationConfiguration(MethodValidationConfiguration mvc);
```

Comment:
Regarding `isAllowMultipleCascadedValidationOnReturnValues()` etc., what's the exact semantics of these? Do they return the current value - i.e. including any changes applied by calling the setters - or does it only reflect the default settings? If so - which I think would make sense to be with the existing API - we should somehow encode the "default" world into the names to be inline with other getters and make the meaning apparent.